ca/server.go

```diff
@@ -740,11 +737,13 @@ func (s *Server) signNodeCert(ctx context.Context, node *api.Node) error {
 	)
 
 	// Try using the external CA first.
-	cert, err := externalCA.Sign(ctx, PrepareCSR(rawCSR, cn, ou, org))
+	s.secConfigMu.Lock()
```

Comment:
I think not, because the test also now waits for the CA to have stabilized before running the actual test assertions.  But I'll check.